tests/unit/aiplatform/test_docker_utils.py

```diff
@@ -676,6 +676,19 @@ def test_build_image(self, make_dockerfile_mock, execute_command_mock):
             pip_command=self.PIP,
             python_command=self.PYTHON,
         )
+        execute_command_mock.assert_called_once_with(
```

Comment:
Is this change to the unit tests still useful?

Reply:
yes, so we make sure the local_utils.execute_command is called as expected. I should have added this in the first place.